pyproject.toml

```diff
@@ -112,14 +115,17 @@ docs = [
     "mkdocs-autorefs",
 ]
 google = ["gspread", "google-api-python-client", "google-auth-oauthlib"]
-oxi = ["oxrdflib"]
-graphql = ["jinja2"]
+oxi = ["oxrdflib", "pyoxigraph"]
+service = ["uvicorn", "prometheus-client", "fastapi", "schedule"]
+
 all = [
-    "gspread",
-    "google-api-python-client",
-    "google-auth-oauthlib",
```

Comment:
I removed these from all as they are only needed for a very specific case. Then, I think the user should install them explicitly `pip install neat[google]`.